Add recommendation to start thinking about data early.

ethanwhite opened the pull request:
This fixes #86. It should not be merged until after #124 (which I branched off from to avoid
a potentially conflicted merge).

Also includes a typo correction in the Acknowledgements.

ethanwhite committed:
Add recommendation to start thinking about data early

sarahsupp commented:
I think the additions are good. One thought - would line 417 (in the data recommendations pull request) still be best as the last sentence? I don't feel strongly either way.

ethanwhite commented:
Yeah, it's definitely better at the end. It was originally our last sentence and I'm not even sure why I moved it. It's back at the end now, plus I replaced the first "practices" with "recommendations" in the last paragraph to aid the transition between it and the penultimate paragraph.
